packages/blade/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -139,7 +138,7 @@ export const ControlledAndUncontrolled = _ControlledAndUncontrolled.bind({});
 
 export const checkboxRef: ComponentStory<typeof CheckboxComponent> = () => {
   // eslint-disable-next-line react-hooks/rules-of-hooks
-  const checkboxRef = React.useRef<BladeElementRef>(null);
+  const checkboxRef = React.useRef<HTMLInputElement>(null);
```

Comment:
Is this not breaking on react native?

Reply:
No won't be a breaking change, underline types are kept wide in purpose so that it doesn't throw any type errors. 

Plus since we removed the whole useBladeInnerRef thing it doesn't make sense for consumers to use BladeElementRef anymore they can, on RN they will do `useRef<View>` or `useRef<Pressable>` etc